lutris/runners/scummvm.py

```diff
@@ -11,7 +11,7 @@
 class scummvm(Runner):
     description = "Runs various 2D point-and-click adventure games."
     human_name = "ScummVM"
-    platform = "2D point-and-click games"
+    platforms = "2D point-and-click games"
```

Comment:
ScummVM should likely be `PC`, the majority of the games were originally released on DOS.